### ISSUE TITLE
Choices

### ISSUE THREAD

**Issue by Yajo** [pykong/copier#93]
Supporting choices in `copier.yml` would be awesome for the UX of cloning. After fixing #89, it could be added like this:

```yaml
license:
  help: Choose a license for your project among the supported ones
  choices:
    - MIT
    - AGPL
    - GPL
  default: GPL
```

Then the user would be prompted to enter a number (from 1 to 3) or the exact content of one of the list elements.

**Comment by pykong** [comment]
This is an idea I would accept a well-formed **PR** for.

**Comment by Yajo** [comment]
This was fixed in #107.
